File: Cafeteria.Shared/DTOs/Menu/FoodOptionTypeDto.cs
using System.ComponentModel.DataAnnotations;

namespace Cafeteria.Shared.DTOs.Menu;

public class FoodOptionTypeDto
{
    public int Id { get; set; }

    [Required]
    [StringLength(50)]
    public string FoodOptionTypeName { get; set; } = string.Empty;

    [Required]
    [Range(0, int.MaxValue)]
    public int RequiredAmount { get; set; }

    [Required]
    [Range(0, int.MaxValue)]
    public int IncludedAmount { get; set; }

    [Required]
    [Range(0, int.MaxValue)]
    public int MaxAmount { get; set; }

    [Required]
    [Range(0, double.MaxValue)]
    public decimal FoodOptionPrice { get; set; }

    public int? EntreeId { get; set; }

    public int? SideId { get; set; }
}

File: ReceiptPrinter/DTOs/FoodOptionTypeDto.py
from pydantic import BaseModel
from typing import Optional
from decimal import Decimal


class FoodOptionTypeDto(BaseModel):
    id: int = 0
    foodOptionTypeName: str = ""
    requiredAmount: int = 0
    includedAmount: int = 0
    maxAmount: int = 0
    foodOptionPrice: Decimal = Decimal("0")
    entreeId: Optional[int] = None
    sideId: Optional[int] = None
    iconId: Optional[int] = None
    iconBootstrapName: Optional[str] = None
